EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRaw.cc

```diff
@@ -25,7 +25,8 @@ namespace sistrip {
     mode_(mode),
     packetCode_(packetCode),
     useFedKey_(useFedKey),
-    bufferGenerator_()
+    bufferGenerator_(),
+    warnings_("DigiToRaw", "[sistrip::DigiToRaw::createFedBuffers_]", edm::isDebugEnabled())
```

Comment:
shouldn't this use a common constant from DataFormats/SiStripCommon/interface/ConstantsForLogger.h
similar to ```sistrip::RawToDigiUnpacker``` ?
... OTOH, it doesn't look like ```mlRawToDigi_``` is used consistently elsewhere with frequent direct ```"SiStripRawToDigi"```.
So, feel free to ignore.

Reply:
I solved it like this to keep the more or less the same warning messages. There is no constant for the packer, and they are used rather inconsistently in several places... but I would leave cleaning that up for another time.